lib/ansible/modules/network/bigswitch/bigmon_policy.py

```diff
@@ -0,0 +1,215 @@
+    state = module.params['state']
+    controller = module.params['controller']
+
+    rest = Rest(module,
+                {'content-type': 'application/json', 'Cookie': 'session_cookie='+access_token},
+                'https://'+controller+':8443/api/v1/data/controller/applications/bigtap')
+
+    if name is None:
+        module.fail_json(msg='parameter `name` is missing')
+
+    response = rest.get('policy?config=true', data={})
+    if response.status_code != 200:
+        module.fail_json(msg="failed to obtain existing policy config: {}".format(response.json['description']))
+
+    config_present = False
+
+    matching = [policy for policy in response.json
+                if policy['name'] == name and
+                   policy['duration'] == duration and
+                   policy['delivery-packet-count'] == delivery_packet_count and
+                   policy['policy-description'] == policy_description and
+                   policy['action'] == action and
+                   policy['priority'] == priority]
+
+    if matching:
+        config_present = True
+
+    if state in ('present') and config_present:
+        module.exit_json(changed=False)
+
+    if state in ('absent') and not config_present:
+        module.exit_json(changed=False)
+
+    if state in ('present'):
+        data={'name': name, 'action': action, 'policy-description': policy_description,
+              'priority': priority, 'duration': duration, 'start-time': start_time,
+              'delivery-packet-count': delivery_packet_count }
+
+        response = rest.put('policy[name="%s"]' % name, data=data)
+        if response.status_code == 204:
+            module.exit_json(changed=True)
+        else:
+            module.fail_json(msg="error creating policy '{}': {}".format(name, response.json['description']))
+
+    if state in ('absent'):
+        response = rest.delete('policy[name="%s"]' % name, data={})
+        if response.status_code == 204:
+            module.exit_json(changed=True)
+        else:
+            module.fail_json(msg="error deleting policy '{}': {}".format(name, response.json['description']))
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            name=dict(type='str', required=True),
+            policy_description=dict(type='str', default=''),
+            action=dict(choices=['forward', 'drop', 'capture', 'flow-gen'], default='forward'),
+            priority=dict(type='int', default=100),
+            duration=dict(type='int', default=0),
+            start_time=dict(type='str', default=datetime.datetime.now().isoformat()+'+00:00'),
+            delivery_packet_count=dict(type='int', default=0),
+            controller=dict(type='str', required=True),
+            state=dict(choices=['present', 'absent'], default='present'),
+            validate_certs=dict(type='bool', default='False'),
+            access_token=dict(aliases=['BIGSWITCH_ACCESS_TOKEN'], no_log=True)
```

Comment:
You've not specified this alias in the documentation. 

If you meant to read from the environment then you need to read from the the standard python environment object.